packages/notification-service/src/blockscout/decode.ts

```diff
@@ -56,19 +56,9 @@ function decodeTransferLog(log: Log): Transfer | null {
     return null
   }
 
-  /**
-   * Decode using the parameter signature for an ERC20 Transfer event
-   * For unknown reasons, blockscout includes an extra unknown param in the log's topics list
-   * Including this unknown param in the input list or decoding won't work
-   */
   try {
-    const decodedLog = web3.eth.abi.decodeLog(
+    const decodedLog = abiCoder.decodeLog(
       [
-        {
```

Comment:
Well this is the change that I've told you about. It's not that blockscout add it an unknown one, it's that apparently, if the event has an owner, the first parameter is the topic of an address, but is not expected to be add that topic to the decoder.
That's why, I'm removing it:
```
// Docs: An array with the index parameter topics of the log, without the topic[0] if its a non-anonymous event.`
log.topics.slice(1).filter(notEmpty)
```
You can check it here: https://web3js.readthedocs.io/en/v1.2.11/web3-eth-abi.html#decodelog

What I think that was happening, is that we added that unknown for the input, because we were adding the first topic, without knowing this owner thing. This way, this first topic, basically was adding garbage to that `unknown` field, and as we didn't use it, it was discarded.

In that version of web3 they were probably not checking if it was valid, but in this case yes. That's why I require to use it as it should, and it's returning the same exactly data as before